config.yaml

```diff
@@ -62,6 +81,32 @@ calendar:
 
 runspersub: 1
```

Comment:
The default is 1, so we could remove this to keep things as simple as possible.

We also have the option of making this a larger number. The [queue limits for `gadi`](https://opus.nci.org.au/display/Help/Queue+Limits) are such that we could run this for 48 hours per submit. This probably equates to 20 year runs per submit (assuming there is a little padding in the 2.5 hour queue limit).

This would require setting `walltime: 48:00:00`.

The downside is when testing and running less than 20 hours it would potentially increase the queueing time because it is asking for more resources (`walltime`).

I've [made an issue to look into changing the default behaviour](https://github.com/payu-org/payu/issues/479), but that won't be done before we release, so we would have to put up with the sub-optimal behaviour if we increased this.

Reply:
Good idea. I'd kept it in originally as the Hive docs currently use it when discussing [ways to change the total simulation lengths](https://access-hive.org.au/models/run-a-model/run-access-esm/#understand-runtime-runspersub-and-n-parameters). I guess there's no reason to have the current Hive docs as the controlling factor. 

With the proposed changes it sounds like we would have a large value like 20 as the default – Do you think it makes sense to set this now (and increase the walltime) so that there are minimal changes when we add the future updates?

Reply:
Ok, scratch this (my) idea. 

We should wait until we have a better solution implemented in `payu`. It's nice and simple if it's just a single run, and if we get complaints about poor throughput we can suggest this as a solution.

Sorry! I know you've made that change already.

Reply:
No worries! I've just switched it back to 1 and reduced the walltime back to 2:30:00